src/components/ServiceDetails/MediaGallery/index.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from "react";
 import { withStyles } from "@material-ui/styles";
 import ImageGallery from "react-image-gallery";
+import last from "lodash/last";
 import "react-image-gallery/styles/css/image-gallery.css";
 
 import { useStyles } from "./styles";
 
-// const PREFIX_URL = "https://raw.githubusercontent.com/xiaolin/react-image-gallery/master/static/";
+const PREFIX_URL = "https://raw.githubusercontent.com/xiaolin/react-image-gallery/master/static/";
```

Comment:
do we need this still??

Reply:
if yes, then we have to get the static image from sujith. We should not use the one that the package provides